docs/api/public_php_api_browsing.md

```diff
@@ -101,7 +101,7 @@ If you want to take SiteAccess languages into account:
 -   Provide prioritized languages on `load()` this will be taken into account by the returned Content object when retrieving translated properties like fields, for example:
     `$contentService->loadContent( 66, $configResolver->getParameter('languages'));`
     -   `ConfigResolver` is a service, so it can be obtained from Symfony Container or injected directly: `@ezpublish.config.resolver`.
-    -   *Note: As of v2.0 this is planned to be done for you when you don't specify languages.*
+    -   See [below](#siteaccess-aware-repository-optional) for information about SiteAccess-aware repository.
```

Comment:
Outdent this, as it's an alternative choice, and maybe start with `Or if you want repository to inject this for you see ..` or something